index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <title>Anime Quiz</title>
+	<link rel="stylesheet" href="stylesheet.css"/>
+</head>
+<body>
+    <div id="app">
+		<div class="background-image"></div>
```

Comment:
I think, that you can move all content to the ReactJS APP.

Reply:
Moved to JSX